lib/helpers.dart

```diff
@@ -5,14 +5,27 @@ import 'package:stingray/history.dart';
 import 'package:stingray/model/item.dart';
 
 void handleShare(int id) {
-  String url = "https://news.ycombinator.com/item?id=$id";
+  String hnUrl = buildHackerNewsURL(id);
 
   String text =
-      "Hey, check out this Hacker News story that I read via Stingray!\r\r $url";
+      "Hey, check out this Hacker News story that I read via Stingray!\r\r $hnUrl";
 
   Share.share(text);
 }
 
+void handleShare(int id, String title, String postUrl) {
```

Comment:
While we're at it, can we turn this into named arguments? Should be as simple as adding curly braces:

`handleShare({int id, String title, String postUrl})`

This would also require the callsite to update the call to look like:

`handleShare(id: item.id, title: item.title, postUrl: item.url)`